### PR TITLE
[MVPN-203] Connection history

### PR DESCRIPTION
**Pull request by donce**
46519803-5c05fc80-c882-11e8-98c5-ba3b7f07c333

Fixes: https://github.com/mysteriumnetwork/mysterium-vpn/issues/405

### CODE REVIEW

File: src/renderer/components/session-item.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    return {
+      dateObject: null
+    }
+  },
+  created: function () {
+    this.dateObject = new Date(this.session.dateStarted)
+  },
+  computed: {
+    durationString () {
+      return formatTimeDisplayOrDefault(this.session.duration)
```
Comment (ignasbernotas) [inline src/renderer/components/session-item.vue]: ?? Shouldn't this function be moved to time formatter class? Now there are 2 ways to format time and I need to check the internals to figure out which one to use:)

File: src/renderer/pages/connection-history.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  name: 'ConnectionHistory',
+  components: { CloseButton, SessionItem },
+  dependencies: ['tequilapiClient'],
+  data: function () {
+    return {
+      sessions: []
+    }
+  },
+  created: function () {
+    this.tequilapiClient.sessionsList().then(sessions => {
```
Comment (ignasbernotas) [inline src/renderer/pages/connection-history.vue]: 👍 So nice to see tequilapi calls decoupled from vuex 😄

File: src/libraries/time-formatter.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  _getDateWithOffset (date: Date): Date {
+    const newDate = new Date(date.getTime())
+    newDate.setMinutes(newDate.getMinutes() - this._minutesOffset)
+    return newDate
+  }
+}
+
+const FORMAT_LOCALE = 'en-GB'
```
[the remark this answers is not in the document]
Comment (donce) [reply src/libraries/time-formatter.js]: I think that's what we decided with @AndriusMyst

File: src/renderer/components/session-item.vue (hunk 1 above)
Comment (donce) [reply src/renderer/components/session-item.vue]: Extracted separate classes: `TimeFormatter`, `DurationFormatter` and `BytesFormatter`

**Pull request merged**